src/main/java/cceclipseplugin/core/CCIgnore.java

```diff
@@ -0,0 +1,132 @@
+package cceclipseplugin.core;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Path;
+
+import cceclipseplugin.ui.dialogs.MessageDialog;
+import dataMgmt.MetadataManager;
+import dataMgmt.models.FileMetadata;
+import websocket.models.responses.FileCreateResponse;
+
+public class CCIgnore {
+	
+	public static String FILENAME = ".ccignore";
+	public static String DEFAULT_CONTENTS = "CodeCollaborateConfig.json\nbin\n";
```

Comment:
I feel like this should be laid out, maybe have just an actual default physical file that gets copied into a project directory on creation. that would make it a lot easier for us to make changes to it.